Auth0/CredentialsManager.swift

```diff
@@ -80,4 +82,11 @@ public struct CredentialsManager {
             }
         }
     }
+
+    public func credentialsOnTouch(withTitle title: String, scope: String? = nil, callback: @escaping (CredentialsManagerError?, Credentials?) -> Void) {
```

Comment:
Can't we keep the other interface and make TouchID a configuration thing of the credential manager?